rental.rb

```diff
@@ -1,13 +1,15 @@
+require './date'
```

Comment:
- Kindly use the appropriate form when using require  `ex: require 'date' `it will fix the error you are facing when running your main.rb file  :+1: :+1: :+1: